bindings/cs/MongoDB.Libmongocrypt/CryptClientFactory.cs

```diff
@@ -109,6 +109,7 @@ public static CryptClient Create(CryptOptions options)
                 }
 
                 Library.mongocrypt_setopt_use_need_kms_credentials_state(handle);
+                Library.mongocrypt_setopt_fle2v2(handle, true);
```

Comment:
let's configure it as the first step after new as it's done in c impl: https://github.com/mongodb/mongo-c-driver/commit/8aced03a963e70182b79d883e881f239766b09bf#diff-6611c8b7be663c2fc9c7942692d314e8464f9398d0275e7ce691905fd5eeeeb0R1330

Reply:
Also, you need to analyze method result, see `mongocrypt_setopt_encrypted_field_config_map` call above as example

Reply:
I mean `handle.Check(` part

Reply:
Done.